app/views/user_mailer/second_upload_reminder.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= stylesheet_link_tag 'emails', media: 'all' %>
 
-<table class='second-upload-reminder-email'>
+<table class='third-upload-reminder-email'>
```

Comment:
This surprised me - can you help me understand what's up here?

Reply:
I renamed `third_upload_reminder.html.erb` to `second_upload_reminder.html.erb` after deleting the original `second`, but I didn't change any of the contents. I will, though - I've changed "third" to "second" everywhere else, so this will be confusing. Good catch!

Reply:
Should be fixed in a059027!